### DISCUSSION TITLE
Docker-Wyze-Bridge Home Assistant Add-On Discontinued?

### DISCUSSION THREAD

**Discussion by CynicalSaint48** [mrlt8/docker-wyze-bridge#1254] [General]
My latest update of Home Assistant resulted in the automatically deletion of the Docker-Wyze-Bridge, and I am unable to find it or any replacement.  

Is this another case of Wyze violently opposing Home Assistant and Home Kit integrations?

Or is there a known workaround?

Thanks in advance!

**Comment by mrlt8** [comment]
Still working for me. Were you using one of the dev or other builds? You may need to enable advanced mode in your user profile page to see those.

**Comment by CynicalSaint48** [comment]
Ok, further investigated and found several 'my problem' type issues.  Heh

I am on a very old version, 2.3.17;  I assume that it what is causing the error:

![dock-wyze-bridge_HA](https://github.com/mrlt8/docker-wyze-bridge/assets/47509013/246d2ebc-f5dd-4b95-a274-07c7404a6f1a)

For the moment, I have not allowed it to delete the add-on, but that is what I need to do; delete and update.  

Finally, it did not fully 'break', wyze bridge is still running in the Supervisor, and viewable directly in the Add-On, but the linked camera feeds on one of my 'Views', are what have stopped working.  

A wise person once said, 'If it doesn't work after reboot, then it's probably user error.'  In this case, true.  Heh

**Reply by mrlt8** [reply]
FYI, wyze recently disabled their authentication endpoints, so you will probably need to update your config to include those if/when your refresh tokens expire.

**Comment by jeffalperin** [comment]
I'm getting the same message about docker wyze bridge being removed from the repository, but I'm already running the latest version 2.9.7.  The add-on appears to be working just fine, so maybe it's just the error message that erroneous?

**Comment by rsilves** [comment]
Hi everyone. I had this issue, I was also on 2.3.17, I uninstalled and reinstalled the newer version and the error persisted, then I uninstalled, restarted and installed the new version and the issue message disappeared.
